src/styles/index.scss

```diff
@@ -1,3 +1,220 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+  --column-count: 1;
+}
+
+.imgblock {
+  height: 40px;
+  width: 40px;
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  color: inherit;
+}
+
+nav {
+  align-content: center;
+}
```

Comment:
The 'nav' element is styled with 'align-content: center;', but this property is not effective for flex containers. Consider using 'align-items: center;' instead.